packages/account-management/src/hooks/use-location-query.tsx

```diff
@@ -0,0 +1,10 @@
+import { useLocation } from 'react-router-dom';
```

Comment:
This file (and other hooks) should probably live in the `frontend` folder. The Node server has no use for React hooks.